x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/Security.java

```diff
@@ -1332,6 +1354,12 @@ private static XContentBuilder getIndexMappings() {
                     builder.field("dynamic", false);
                     builder.endObject();
 
+                    if (Version.V_7_13_0.onOrBefore(mappingVersion)) {
```

Comment:
```suggestion
                    if (mappingVersion.onOrAfter(Version.V_7_13_0)) {
```
I think it reads more clearly that way